src/components/Collisions.ts

```diff
@@ -380,6 +383,10 @@ export class Collisions<TGameStartr extends FullScreenPokemon> extends GeneralCo
             return false;
         }
 
+        if (thing.top === other.bottom) {
```

Comment:
Put a todo comment here linking to a new issue mentioning horizontal ledges? Gen II will care.